.gitlab-ci.yml

```diff
@@ -83,6 +88,21 @@ build-in-docker-static:
   variables:
     EXTRA_CMAKE_FLAGS: "-DBUILD_SHARED_LIBS=OFF"
 
+upload-artifacts:
+  stage: upload
+  extends: .in-prplmesh-builder
+  script:
+    - ln -s "$CI_OWNCLOUD_PASSWORD" ~/.netrc
+    - sed -i 's/^/machine ftp.essensium.com login prplmesh-robot-ci password /' ~/.netrc
```

Comment:
This weirdness warrants some comment to explain what is going on. Even I am not sure I understand it - you symlink a file (which I suppose is some file that exists on the runner) and then you do a sed -i which IIUC makes a copy of the file...